src_extra/networklayer_tcp.c

```diff
@@ -35,7 +35,6 @@
 
 /* workaround a glibc bug where an integer conversion is required */
 #if !defined(_WIN32)
-# include <features.h>
```

Comment:
It isn't used (as far as I could see) and this file doesn't exist on OS X (it caused compile errors)